### PR TITLE
fix: cta clickability issue in adev homepage.

### LINKED ISSUE #52900: Learn Angular button is clickable even when not visible
### Describe the problem that you experienced

image
In the new angular.dev doc site there is a `Learn Angular` button which is not visible after the first page scroll but stays clickable throughout the main page and obstructs the editor's space and stays on top.
image

### Enter the URL of the topic with the problem

https://angular.dev/

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

It should be removed from the DOM once the first page scroll is done and not be clickable

### Add a screenshot if that helps illustrate the problem

_No response_

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

```true
Instead of changing the opacity to 0, I think we can add an @if/*ngIf condition to show/hide the button
```

### PR DESCRIPTION
**Pull request by SahulKola**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
- Fix Clickability issue

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

- CTA is clickable after the page scroll.

Issue Number: fixes #52900  

## What is the new behavior?
- added `visibility: visible` by default, and `visibility: hidden` property on page scroll

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/app/features/home/home.component.scss [hunk 1]
```diff
@@ -142,7 +142,8 @@
     display: inline-block;
     padding: 7px;
     opacity: 0;
```
Comment (jsuryakt) [inline adev/src/app/features/home/home.component.scss:144]: Maybe we can remove opacity here and for .adev-arrow and use only visibility. WDYT?

Comment (JeanMeche) [reply adev/src/app/features/home/home.component.scss:144]: We use it for the transition.

Comment (jsuryakt) [reply adev/src/app/features/home/home.component.scss:144]: > We use it for the transition. 

Can't we have the transition on visibility?

File: adev/src/app/features/home/home.component.scss [hunk 2]
```diff
@@ -142,7 +142,8 @@
...
-    transition: opacity 0.5s linear;
+    visibility: hidden;
+    transition: opacity 0.5s linear, visibility 0.5s linear;
```
Comment (jsuryakt) [inline adev/src/app/features/home/home.component.scss:146]: If transition doesn't work on visibility, we can remove it since we already have opacity.

Comment (JeanMeche) [reply adev/src/app/features/home/home.component.scss:146]: no, visibility prevents from being able to interact with it. So we're fine.

**Review by jessicajaniuk — APPROVED** [review]
Verified this works. Looks great! Thank you!

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit dfc6c8d0c76380ee7beb6d904261e40857b375ed.

**Pull request closed without merging**